staging/src/k8s.io/client-go/openapi/client.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openapi
+
+import (
+	"context"
+	"encoding/json"
+	"sync"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/kube-openapi/pkg/handler3"
+)
+
+type Client interface {
```

Comment:
I love that non-smurfy name, should it say v3 somewhere though?

Reply:
I thought adding V3 to either package or the type, but decided against it considering v2 is going away. I'll leave it to your discretion, though. I am 50/50 on which way to go.

Reply:
Yeah, I guess I'm fine with that argument